### PR TITLE
dezoomify-rs 2.11.2 (new formula)

### PR DESCRIPTION
**Pull request by kostas**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> shasum -c testtiles_shasum.txt
  shasum: WARNING: 1 computed checksum did NOT match
  testtile.png: FAILED
```

File: Formula/d/dezoomify-rs.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class DezoomifyRs < Formula
+  desc "Tiled image downloader"
+  homepage "https://github.com/lovasoa/dezoomify-rs"
+  url "https://github.com/lovasoa/dezoomify-rs/archive/refs/tags/v2.11.2.tar.gz"
+  sha256 "19bcd079d8a370456080ba65b2f51fa4aa2eee00207a18aa9782dd7faef9acb9"
+  license "GPL-3.0-only"
+
+  head do
+    url "https://github.com/lovasoa/dezoomify-rs.git", branch: "master"
+  end
```
Comment (p-linnane) [inline Formula/d/dezoomify-rs.rb:10]:
```suggestion
  head "https://github.com/lovasoa/dezoomify-rs.git", branch: "master"
```

Comment (p-linnane) [reply Formula/d/dezoomify-rs.rb:10]: We only need a `head` block if there are HEAD only dependencies.

**Review by p-linnane — CHANGES_REQUESTED** [review]

**Comment by p-linnane** [comment]
After addressing the review please squash your commits into one with the message `dezoomify-rs 2.11.2 (new formula)`. Thanks.

File: Formula/d/dezoomify-rs.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class DezoomifyRs < Formula
+  desc "Tiled image downloader"
+  homepage "https://github.com/lovasoa/dezoomify-rs"
+  url "https://github.com/lovasoa/dezoomify-rs/archive/refs/tags/v2.11.2.tar.gz"
+  sha256 "19bcd079d8a370456080ba65b2f51fa4aa2eee00207a18aa9782dd7faef9acb9"
+  license "GPL-3.0-only"
+
+  head "https://github.com/lovasoa/dezoomify-rs.git", branch: "master"
```
Comment (p-linnane) [inline Formula/d/dezoomify-rs.rb:8]:
```suggestion
  head "https://github.com/lovasoa/dezoomify-rs.git", branch: "master"
```

**Review by p-linnane — DISMISSED** [review]
Thanks!

**Pull request closed without merging**